src/plugins/ui_core_views/evaluation_conditional_format.ts

```diff
@@ -91,51 +91,55 @@ export class EvaluationConditionalFormatPlugin extends UIPlugin {
   private getComputedStyles(sheetId: UID): ComputedStyles {
     const computedStyle: ComputedStyles = {};
     for (let cf of this.getters.getConditionalFormats(sheetId).reverse()) {
-      switch (cf.rule.type) {
-        case "ColorScaleRule":
-          for (let range of cf.ranges) {
-            this.applyColorScale(sheetId, range, cf.rule, computedStyle);
-          }
-          break;
-        case "CellIsRule":
-          const formulas = cf.rule.values.map((value) =>
-            value.startsWith("=") ? compile(value) : undefined
-          );
-          for (let ref of cf.ranges) {
-            const zone: Zone = this.getters.getRangeFromSheetXC(sheetId, ref).zone;
-            for (let row = zone.top; row <= zone.bottom; row++) {
-              for (let col = zone.left; col <= zone.right; col++) {
-                const predicate = this.rulePredicate[cf.rule.type];
-                const target = { sheetId, col, row };
-                const values = cf.rule.values.map((value, i) => {
-                  const compiledFormula = formulas[i];
-                  if (compiledFormula) {
-                    return this.getters.getTranslatedCellFormula(
-                      sheetId,
-                      col - zone.left,
-                      row - zone.top,
-                      {
-                        ...compiledFormula,
-                        dependencies: compiledFormula.dependencies.map((d) =>
-                          this.getters.getRangeFromSheetXC(sheetId, d)
-                        ),
-                      }
+      try {
```

Comment:
What exactly throws  in this method ? The `compile` ? In that case we have the same problem with data validation. 

The actual evaluation won't throw since it's done with `getters.evaluateFormula` that have a try/catch

Reply:
It is indeed broken for the data validations as well :) I think we could do the same thing I did for this: 1 fix PR and 1 imp to capture these errors